HoloPipelines/pipelines/components/compDicom2numpy.py

```diff
@@ -47,7 +47,7 @@ def loadPixelArray(inputPath):
 def resample(dataPath, new_spacing=[1, 1, 1]):
     scan = loadScan(dataPath)
     image = loadPixelArray(dataPath)
-    print("Shape before resampling\t", image.shape)
+    logging.info("Shape before resampling\t", image.shape)
```

Comment:
Just found out that this generates the following error when used. 

```
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib64/python3.7/logging/__init__.py", line 1034, in emit
    msg = self.format(record)
  File "/usr/lib64/python3.7/logging/__init__.py", line 880, in format
    return fmt.format(record)
  File "/usr/lib64/python3.7/logging/__init__.py", line 619, in format
    record.message = record.getMessage()
  File "/usr/lib64/python3.7/logging/__init__.py", line 380, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "pipelines/pipelineDicom2glb.py", line 31, in <module>
    main(sys.argv[1], sys.argv[2], sys.argv[3])
  File "pipelines/pipelineDicom2glb.py", line 13, in main
    generatedNumpyList = compDicom2numpy.main(str(pathlib.Path(dicomFolderPath)))
  File "/home/boonwenjie/workspace/COMP0111_SUMMER_PROJ/holo-repo-clean/HoloPipelines/pipelines/components/compDicom2numpy.py", line 97, in main
    imgs_after_resamp, spacing = resample(dicomPath)
  File "/home/boonwenjie/workspace/COMP0111_SUMMER_PROJ/holo-repo-clean/HoloPipelines/pipelines/components/compDicom2numpy.py", line 52, in resample
    logging.info("Shape before resampling\t", image.shape)
Message: 'Shape before resampling\t'
Arguments: ((160, 512, 512),)
```

Reply:
A fix will be to use the repr version of `image.shape` and form the log message. 
```suggestion
    logging.info("Shape before resampling\t" +  repr(image.shape))
```

Reply:
This logging style is found 4 times in the project. Please change them all. 
```
./pipelines/components/compNumpy2obj.py:    logging.info(image.shape)
./pipelines/components/compDicom2numpy.py:    logging.info("Shape before resampling\t" + repr(image.shape))
./pipelines/components/compDicom2numpy.py:    logging.info("Shape after resampling\t" + repr(image.shape))
./pipelines/components/compNifti2numpy.py:    logging.info("Shape after resampling\t", image.shape[:3])
```

Reply:
Update all `image.shape` with repr version. (removed `logging.info(image.shape)` from compNumpy2obj)